spec/unit/puppet/provider/scheduled_task/win32_taskscheduler_spec.rb

```diff
@@ -655,11 +655,9 @@ def time_component
 
   describe '.instances' do
     it 'should use the list of .job files to construct the list of scheduled_tasks' do
```

Comment:
Test name should me modified as `.job` is no longer part of the test

Reply:
Done